### ISSUE TITLE
Why emails are not getting send to other mail server

### ISSUE THREAD

**Issue by rover886** [docker-mailserver/docker-mailserver#2477]
# Subject

I am trying to send email to other domain For eg. gmail.com, but it was not getting send.

## Description

From this docker mail server I want to send email to gmail.com but it couldn't get send. I see below logs appearing whenever I tried sending email to other domain

```
Mar 11 12:45:21 mail postfix/smtp[26783]: connect to gmail-smtp-in.l.google.com[74.125.24.27]:25: Connection refused
<example@gmail.com>, relay=none, delay=106, delays=0.01/0.01/106/0, dsn=4.4.1, status=deferred (connect to alt4.gmail-smtp-in.l.google.com[142.250.115.27]:25: Connection refused)  
```

IP address of respective email server is getting resolved, may b docker is not able to connect on port 25, so how can I make it working?

**Comment by casperklein** [comment]
This is probably not related to DMS. Check the connectivity of your server. Some providers block tcp/25.
If that's the case, not only Gmail is affected. You won't be able to send mails to other domains as well.

**Comment by polarathene** [comment]
If your server is not able to send mail to others on port 25, you will need to relay through a service like SendGrid via a port that is not blocked. Alternatively choose a server that is not blocked, usually this costs more and is less convenient which makes it less attractive to spammers to abuse.

I am pretty sure RamNode allows you to use port 25, but if the IP block your server address is assigned has a bad reputation from spammers it may still be an issue. Vultr also allows port 25 IIRC, but it's blocked by default requiring you to request it explicitly through support, although I'm not familiar with the process and what is required if anything for approval.

**Comment by rover886** [comment]
Yes @casperklein gmail.com is just an example I have taken, I could not be able to send mail to any domain.

**Comment by rover886** [comment]
@polarathene , Right now I am using `RELAY_HOST` to send email to outside domain. And about the options you have given those might work but I don't want to spent time on those because I am using this docker mail server as a test server in my pipeline. Using `RELAY_HOST` is not the solution because in that case `From` header gets modified to relay user. So, I need some quick way to resolve this. This will never be production mail server. I am still curious how this [docker mailserver](https://github.com/jeboehm/docker-mailserver) manages to send email to other domain?

**Comment by polarathene** [comment]
> So, I need some quick way to resolve this. This will never be production mail server.

If you're not using this for a completely offline/local test environment and your only intention is to send a small amount of emails from your setup to an external provider, perhaps DMS is not your best choice and you should use one of the relay services with their APIs? Often they provide a free tier that can send a low amount of emails per month.

If you avoid a relay service such as SendGrid, you need to ensure your server/IP has a good reputation to successfully deliver, and you must also ensure that your server is able to send email out on port 25 (or 587/465 for relay submission), if using an online service that port is probably blocked for outgoing traffic. If your server is a local system, the ISP may be blocking it.

**Comment by rover886** [comment]
Thanks for suggestions. Closing this issue.
